packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -122,6 +122,30 @@ abstract class FlutterCommand extends Command<Null> {
     _usesPubOption = true;
   }
 
+  /// Adds flags for using a specific filesystem root and scheme.
+  ///
+  /// Only usable in Dart 2 mode.
+  ///
+  /// [hide] indicates whether or not to hide these options when the user asks
+  /// for help.
+  void usesFilesystemOptions({@required bool hide}) {
+    argParser..addOption('output-dill',
```

Comment:
formatting looks inconsistent - ..addOption, ..addMultiOption on new lines.

Reply:
Updated to be the same way it's done in attach.dart.